stream-chat-android-core/src/main/java/io/getstream/chat/android/client/call/Call.kt

```diff
@@ -71,8 +71,11 @@ public interface Call<T : Any> {
  *
  * Does not throw exceptions. Any errors will be wrapped in the [Result] that's returned.
  */
-public suspend fun <T : Any> Call<T>.await(): Result<T> =
-    suspendCancellableCoroutine { continuation ->
+public suspend fun <T : Any> Call<T>.await(): Result<T> {
+    if (this is CoroutineCall<T>) {
+        return this.runnable.invoke()
```

Comment:
Not very clear name for `runnable`. Firstly, I thought it is java's `Runnable` and such invoke will blocking. But then looked at implementation details 🙂

Reply:
`task`? `block`? `stuffToRunInACoroutine`?

Reply:
`task`, `block` are  better

Reply:
`blockingTask`? :thinking:

Reply:
It's not blocking, it's suspending! That's the point! 😄

I think `suspendingTask` is gonna be the winner.